engine/apps/slack/views.py

```diff
@@ -29,45 +28,28 @@
 from apps.slack.scenarios.paging import STEPS_ROUTING as DIRECT_PAGE_ROUTING
 from apps.slack.scenarios.profile_update import STEPS_ROUTING as PROFILE_UPDATE_ROUTING
 from apps.slack.scenarios.resolution_note import STEPS_ROUTING as RESOLUTION_NOTE_ROUTING
-from apps.slack.scenarios.scenario_step import (
-    EVENT_SUBTYPE_BOT_MESSAGE,
-    EVENT_SUBTYPE_MESSAGE_CHANGED,
-    EVENT_SUBTYPE_MESSAGE_DELETED,
-    EVENT_TYPE_APP_MENTION,
-    EVENT_TYPE_MESSAGE,
-    EVENT_TYPE_MESSAGE_CHANNEL,
-    EVENT_TYPE_SUBTEAM_CREATED,
-    EVENT_TYPE_SUBTEAM_MEMBERS_CHANGED,
-    EVENT_TYPE_SUBTEAM_UPDATED,
-    EVENT_TYPE_USER_CHANGE,
-    EVENT_TYPE_USER_PROFILE_CHANGED,
-    PAYLOAD_TYPE_BLOCK_ACTIONS,
-    PAYLOAD_TYPE_DIALOG_SUBMISSION,
-    PAYLOAD_TYPE_EVENT_CALLBACK,
-    PAYLOAD_TYPE_INTERACTIVE_MESSAGE,
-    PAYLOAD_TYPE_MESSAGE_ACTION,
-    PAYLOAD_TYPE_SLASH_COMMAND,
-    PAYLOAD_TYPE_VIEW_SUBMISSION,
```

Comment:
these constants have been moved to `enum.StrEnum`s instead. See [here](https://github.com/grafana/oncall/pull/2653/files#diff-fff7dbf78c46322b461b233ca25819802cb10101067580f6bb1763b14a02ac5bR8-R161)